compliance_suite/functions/requestor.py

```diff
@@ -0,0 +1,71 @@
+import json
```

Comment:
Perhaps more standard to call this module `client.py` rather than `requestor.py`?